dubbo-xds/pom.xml

```diff
@@ -84,6 +84,18 @@
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcprov-ext-jdk15on</artifactId>
         </dependency>
+        <dependency>
```

Comment:
in dubbo-parent has org.junit.jupiter

Reply:
Thank you, chengming, I have changed this.